README.md

```diff
@@ -117,6 +118,56 @@ k8s_deploy(
 )
 ```
 
+### Multi-Object Actions
+
+It is common practice in the Kubernetes world to have multiple objects that
+comprise an application.  There are two main ways that we support interacting
+with these kinds of objects.
+
+The first is to simply use a template file that contains your N objects
```

Comment:
Is this basically a multi-document YAML file, or are we doing anything special here?

Reply:
Just a multi-part yaml, nothing fancy.